pytest_inmanta/plugin.py

```diff
@@ -186,6 +188,7 @@ def project_no_plugins(
     DATA.clear()
     project_shared_no_plugins.clean()
     project_shared_no_plugins.init(capsys)
+    project_shared_no_plugins.synchronize_interpreters()
```

Comment:
Let's move this call into the `init()` method. Then the `init()` and the `clean()` method contain complementary logic.